tests/pl/weather_HassGetWeather.yaml

```diff
@@ -0,0 +1,25 @@
+language: pl
+tests:
+  - sentences:
+      - "jaka jest pogoda"
+    intent:
+      name: HassGetWeather
+    response: 8 °C i pada deszcz
+
+  - sentences:
+      - "jaka jest pogoda w Warszawa"
+      - "jaka jest pogoda dla Warszawa"
+
+    intent:
+      name: HassGetWeather
+      slots:
+        name: Warszawa
+    response: 24 °C bez zachmurzenia
+
+  - sentences:
+      - "jaka jest pogoda w Poznań"
```

Comment:
The correct sentence should be "jaka jest pogoda w Poznaniu"

Reply:
see above, this would need aliases or automatic declination. I'll see if I can set up aliases in test fixtures

Reply:
ok, added declinated versions together with the base version also if, I think we should test for most possibilities than can happen (although right now it's just a dumb slot replacement)